docs/api/covidcast-signals/dsew-cpr.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Data Strategy and Execution Workgroup Community Profile Report
+parent: Data Sources and Signals
+grand_parent: COVIDcast Epidata API
+---
+
+# Data Strategy and Execution Workgroup Community Profile Report (CPR)
+{: .no_toc}
+
+* **Source name:** `dsew-cpr`
+* **Earliest issue available:** 2022-01-28
+* **Number of data revisions since 19 May 2020:** 0
+* **Date of last change:** Never
+* **Available for:** county, msa, state, hhs, nation (see [geography coding docs](../covidcast_geography.md))
+* **Time type:** day (see [date format docs](../covidcast_times.md))
+* **License:** [Open Data Commons Open Database License (ODbL)](https://opendatacommons.org/licenses/odbl/1-0/)
+
+The Community Profile Report (CPR) is published by the Data Strategy and Execution Workgroup (DSEW) of the White House COVID-19 Team. For more information, see the [official description and data dictionary at healthdata.gov](https://healthdata.gov/Health/COVID-19-Community-Profile-Report/gqxm-d9w9) for "COVID-19 Community Profile Report".
+
+This data source provides various COVID-19 related metrics, of which we report hospital admissions.
+County, MSA, state, and HHS-level values are pulled directly from CPR; nation-level values are aggregated up from the state level.
+
+| Signal | Description |
+| --- | --- |
+| `confirmed_admissions_covid_1d_7dav` | Seven-day average of adult and pediatric confirmed COVID-19 hospital admissions occurring each day.
+Earliest date available: 2019-12-16 for state, HHS, and nation; 2021-01-06 for MSA and county |
+
+## Table of contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+
+## Estimation
+
+For counts-based fields like hospital admissions, CPR reports rolling sums for the preceding 7 days. The 7-day average signals are computed by Delphi by dividing each sum by 7 and assigning it to the last date in the included range, so e.g. the signal for June 7 is the average of the underlying data for June 1 through 7, inclusive.
+
+The `confirmed_admissions_covid_1d_7dav` signal mirrors the `Confirmed COVID-19 admissions - last 7 days` CPR field for all geographic resolutions except nation. Nation-level admissions is calculated by summing state-level values.
+
+## Limitations
+
+Nation-level estimates may be inaccurate since aggregations are done using state-level smoothed values instead of raw values. Ideally we would aggregate raw values before smoothing, but the raw values are not accessible in this case.
+
+Because DSEW does not provide updates on weekends, estimates are not available for all dates.
+
```

Comment:
```suggestion

### Differences with HHS reports

An analysis comparing the 
[CPR labeled January 5, 2022](https://healthdata.gov/api/views/gqxm-d9w9/files/14ee1150-edf1-4b54-b225-500c8954e6a8?download=true&filename=Community%20Profile%20Report%2020220105.xlsx) 
(newest file as of January 6, 2022) with the HHS 
[COVID-19 Reported Patient Impact and Hospital Capacity by State Timeseries](https://healthdata.gov/Hospital/COVID-19-Reported-Patient-Impact-and-Hospital-Capa/g62h-syeh) 
(downloaded January 6, 2022) suggests that the CPR report undercounts the hospital admissions published by HHS by ~10-15%.

A copy of this analysis is available in Python Notebook format. We are waiting from clarification from the data provider, but until then, exercise caution when comparing work based on the CPR with work based on HHS reports.




```

Reply:
The 10-15% statistic doesn't hold depending on which columns/locations you compare, so I think it is more accurate to say "CPR typically reports lower counts of hospital admissions than those published by HHS". 

I don't see a problem linking the notebook titled `check_cpr_vs_hhs` on Slack, but also don't know if it is necessary.

Reply:
I do want to give folks some idea of scale; how's "10-15% or more" 

?

Reply:
Sounds fine to me.